### PR TITLE
Fix quality a1

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Documentation (non-breaking change which adds documentation)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently the logo is pixelated and not great quality.
- Currently the texts are a bit bold and seem unconnected to the page.

**What is the new behavior (if this is a feature change)?**
- Our logo has been replaced with a SVG logo.
- Some more content has been added.

### CODE REVIEW

**Review by Aichnerc — COMMENTED** [review]
LGTM 🍖

File: src/components/pages/LandingPage/index.jsx [hunk 1]
```diff
@@ -4,11 +4,11 @@ import React from "react";
 
 //> MDB
 // "Material Design for Bootstrap" is a great UI design framework
```
Comment (kleberbaum) [inline src/components/pages/LandingPage/index.jsx:6]: Not so snektonic but still fine. ^^

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Review by LucaAllmaier — APPROVED** [review]
Looks good.

**Pull request merged**